Configuration/PyReleaseValidation/python/relval_standard.py

```diff
@@ -278,6 +278,7 @@
 workflows[1325.1] = ['', ['TTbar_13','DIGIUP15','RECOUP15_trackingOnly','HARVESTUP15_trackingOnly']]
 workflows[1325.2] = ['', ['TTbar_13','DIGIUP15','RECOUP15_trackingLowPU','HARVESTUP15']]
 workflows[1325.3] = ['', ['TTbar_13','DIGIUP15','RECOUP15_trackingOnlyLowPU','HARVESTUP15_trackingOnly']]
+workflows[1325.4] = ['', ['TTbar_13','DIGIUP15','RECOUP15_HIPM','HARVESTUP15']]
```

Comment:
the goal of this PR should be that all 2015 and 2016 data up to 2016F has the mitigation applied.
This is in line with the rereco plans.